crates/oxc_sourcemap/src/sourcemap_visualizer.rs

```diff
@@ -144,8 +125,9 @@ impl<'a> SourcemapVisualizer<'a> {
                     if ch == '\r' && content.chars().nth(i + 1) == Some('\n') {
                         continue;
                     }
-                    tables.push(content[line_byte_offset..i].encode_utf16().collect::<Vec<_>>());
-                    line_byte_offset = i;
+                    tables
+                        .push(content[line_byte_offset..i + 1].encode_utf16().collect::<Vec<_>>());
+                    line_byte_offset = i + 1;
```

Comment:
Previously new line was at the first character of each table row and I assumed that's a bug.